react-components/src/apps/admin/AllStats/AppStatsComponents/datePicker.js

```diff
@@ -38,4 +36,13 @@ function DatePicker(props) {
     );
 }
 
+DatePicker.propTypes = {
+    presenter: PropTypes.shape({
+        handleStartDateChange: PropTypes.func.isRequired,
+        handleEndDateChange: PropTypes.func.isRequired,
+    }),
+    selectedStartDate: PropTypes.string.isRequired,
+    selectedEndDate: PropTypes.string.isRequired,
```

Comment:
These `selectedStartDate` and `selectedEndDate` props should probably be required date objects, rather than required strings.